### PR TITLE
Arangodb 3.1.x

### PR DESCRIPTION
**Pull request by m0ppers**

### CODE REVIEW

File: repo/packages/A/arangodb3/5/config.json [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+          "type": "boolean",
+          "default": true
+        },
+
+        "arangodb-ssl-keyfile": {
+          "description": "Enable SSL using this base64 encoded SSL keyfile (see ArangoDB docs for keyfile specification)",
+          "type": "string",
+          "default": ""
+        },
+        "arangodb-enterprise-key": {
```
Comment (ryadav88) [inline repo/packages/A/arangodb3/5/config.json]: please add a note about `arangodb-ssl-keyfile` and `arangodb-enterprise-key` and how to use them in dcos/examples for arangoDB

File: repo/packages/A/arangodb3/5/command.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (ryadav88) [inline repo/packages/A/arangodb3/5/command.json]: As of packagingVersion 3.0, `command.json` is deprecated in favor of `.cli` property in resource.json 
https://github.com/mesosphere/universe#commandjson

File: repo/packages/A/arangodb3/5/package.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "packagingVersion": "3.0",
+  "name": "arangodb3",
+  "version": "3.1.x",
+  "scm": "https://github.com/arangodb/arangodb3-dcos.git",
+  "maintainer": "info@arangodb.com",
```
Comment (ryadav88) [inline repo/packages/A/arangodb3/5/package.json]: add `minDcosReleaseVersion`
https://github.com/mesosphere/universe#mindcosreleaseversion

File: repo/packages/A/arangodb3/5/config.json (hunk 1 above)
Comment (ryadav88) [inline repo/packages/A/arangodb3/5/config.json]: I created an issue there for tracking purposes: https://github.com/dcos/examples/issues/19

Comment (m0ppers) [inline repo/packages/A/arangodb3/5/config.json]: sent you a note in slack..I think we should either have all options documented or none (these would be the first).

File: repo/packages/A/arangodb3/5/command.json (hunk 2 above)
Comment (m0ppers) [inline repo/packages/A/arangodb3/5/command.json]: done

File: repo/packages/A/arangodb3/5/package.json (hunk 3 above)
Comment (m0ppers) [inline repo/packages/A/arangodb3/5/package.json]: done

**Comment by m0ppers** [comment]
We had some last minute changes and added a few more options :)

File: repo/packages/A/arangodb3/5/package.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+  "packagingVersion": "3.0",
+  "minDcosReleaseVersion": "1.8",
+  "name": "arangodb3",
+  "version": "3.1.x",
+  "scm": "https://github.com/arangodb/arangodb3-dcos.git",
+  "maintainer": "info@arangodb.com",
+  "description": "A distributed free and open-source database with a flexible data model for documents, graphs, and key-values. Build high performance applications using a convenient SQL-like query language or JavaScript extensions.",
```
Comment (ryadav88) [inline repo/packages/A/arangodb3/5/package.json]: please add a link to the example at the end of `description`: Ex:
`Installation documentation can be found at https://github.com/dcos/examples/tree/master/1.8/arangodb`

**Pull request merged**